salt/journal/critical-css.sls

```diff
@@ -49,17 +49,26 @@ journal-local-demo-parameters:
         - require:
             - journal-local-demo-separate-folder
 
+journal-local-demo-critical-css:
+    file.directory:
+        - name: /srv/journal-local-demo/build/critical-css
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - journal-local-demo-separate-folder
+
 journal-local-demo-cache-clean:
     cmd.run:
         - name: bin/console cache:clear --env=demo
         - cwd: /srv/journal-local-demo
         - user: {{ pillar.elife.deploy_user.username }}
         - require:
+            - journal-local-demo-critical-css
             - journal-local-demo-parameters
 
 generate-critical-css:
     cmd.run:
-        - name: echo "We will use gulp to generate critical CSS here"
+        - name: retry "node_modules/.bin/gulp critical-css:generate" 3
```

Comment:
does this also need to retry? I thought it was the image generation that was failing